tests/components/grid.test.ts

```diff
@@ -639,6 +640,56 @@ describe("Grid component", () => {
     });
   });
 
+  describe("filter icon vertical alignment", () => {
+    let centeringOffset: number;
+    let icons: NodeList;
+    beforeEach(async () => {
+      resizeRows(model, [1], DEFAULT_CELL_HEIGHT * 2);
+      createFilter(model, "B2");
+      await nextTick();
+      icons = fixture.querySelectorAll(".o-filter-icon");
+      expect(icons).toHaveLength(1);
+      centeringOffset = (DEFAULT_CELL_HEIGHT - FILTER_ICON_EDGE_LENGTH) / 2;
+    });
+
+    test("filter icon vertical alignment top", async () => {
+      model.dispatch("SET_FORMATTING", {
+        sheetId: model.getters.getActiveSheetId(),
+        target: target("B2"),
+        style: { verticalAlign: "top" },
+      });
+      await nextTick();
+      const top = `${
+        DEFAULT_CELL_HEIGHT * 2 - FILTER_ICON_EDGE_LENGTH + HEADER_HEIGHT - centeringOffset - 1
+      }px`;
```

Comment:
Maybe you happen to get to the right value, but that's not at all the same computations ad we do in `filter_icon_overlay.ts`.... why is there even a `centeringOffset` for the bottom/top align? It doesn't make sense. The computations here should reflect what we do in the codebase.